sources/PseudoTerminal+TouchBar.m

```diff
@@ -522,6 +522,10 @@ - (void)sendFunctionKeyToCurrentSession:(NSInteger)number {
         kVK_F18,
         kVK_F19,
         kVK_F20,
+        NSF21FunctionKey,
```

Comment:
These values aren't actually virtual key codes so they don't belong in this list. There is no virtual keycode for F21-F24 that Apple defines. I suggest putting a value like 0xfff for these with a comment saying "There is no virtual keycode for these function keys, but it doesn't matter for our purposes."

The only place where a bogus virtual keycode would cause problems is in the call to `routeEventToShortcutInputView:` on line 544. Modify that function to return YES but do nothing when the virtual keycode is 0xffff.

Finally, don't use 0xffff as a magic number—define a constant for it in `iTermApplication.h` like:

```
// Used for keys you can press on the touch bar that have no equivalent on physical keyboards that Apple recognizes
extern unsigned short iTermBogusVirtualKeyCode;
````

and in `iTermApplication.m` like

```
unsigned short iTermBogusVirtualKeyCode = 0xffff;
```